.gitlab/build-deb-rpm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Is the intention here to build a `rpm` (or `deb`) with the gems already installed and the dependencies pre-compiled?

Because that usually means that we need to have variants for every Ruby supported, it's usually problematic to pick a native extension that was compiled for some Ruby version and then use it a different Ruby version.